lib/galaxy/managers/group_roles.py

```diff
@@ -25,7 +26,7 @@ class GroupRolesManager:
     def __init__(self, app: MinimalManagerApp) -> None:
         self._app = app
 
-    def index(self, trans: ProvidesAppContext, group_id: EncodedDatabaseIdField) -> List[Dict[str, Any]]:
+    def index(self, trans: ProvidesAppContext, group_id: EncodedDatabaseIdField):
```

Comment:
You can safely hint `GroupRoleListModel` as return type here and `GroupRoleModel` in the rest of the functions that return that model.
```suggestion
    def index(self, trans: ProvidesAppContext, group_id: EncodedDatabaseIdField) -> GroupRoleListModel:
```